### ISSUE TITLE
there is not the file "ckp_denet_83.pth" and "ckp_siknet_synth_41.pth.tar"

### ISSUE THREAD

**Issue by LingSmart** [MengHao666/Minimal-Hand-pytorch#19]
![图片](https://user-images.githubusercontent.com/52542622/122324847-13bc9900-cf5c-11eb-9b69-56dfd191584c.png)
hello,I read read.me carefully ,but i dont find the file i say .

**Comment by MengHao666** [comment]
@LingSmart 
Check every word here more carefully, please. Also it is ok to use pre-trained models u have downloaded, but they were trained without BMC loss.
![image](https://user-images.githubusercontent.com/47114466/122326827-79f6eb00-cf5f-11eb-8a81-9628f8cdb1f1.png)

**Comment by HanXiaoyou** [comment]
I met the save question. The files in Baidu Pan are not inclluded 'checkpoints/ckp_siknet_synth_41.pth.tar' and 'new_check_point/ckp_detnet_83.pth'.
![image](https://user-images.githubusercontent.com/37903733/154643138-28096321-5bcc-4cbe-8fd2-5163b1c1537d.png)
![image](https://user-images.githubusercontent.com/37903733/154643389-69aff6f2-36e6-4703-828b-e7424a30f4c4.png)

**Comment by MengHao666** [comment]
you may refer to  [ShapeNet.md](https://github.com/MengHao666/Minimal-Hand-pytorch/blob/main/ShapeNet.md) for 'checkpoints/ckp_siknet_synth_41.pth.tar'.  As I could remember, 'new_check_point/ckp_detnet_83.pth' is the chekpoint used in real-time demo, that is "bmc_ckp.pth". I am so sorry I didn't describe these things clearly. @HanXiaoyou @LingSmart

**Comment by HanXiaoyou** [comment]
> you may refer to [ShapeNet.md](https://github.com/MengHao666/Minimal-Hand-pytorch/blob/main/ShapeNet.md) for 'checkpoints/ckp_siknet_synth_41.pth.tar'. As I could remember, 'new_check_point/ckp_detnet_83.pth' is the chekpoint used in real-time demo, that is "bmc_ckp.pth". I am so sorry I didn't describe these things clearly. @HanXiaoyou @LingSmart

Thanks！It works.

**Comment by hammerstone18** [comment]
do you mean replace ckp_detnet_83.pth with bmc_ckp.pth?

**Comment by MengHao666** [comment]
> do you mean replace ckp_detnet_83.pth with bmc_ckp.pth?

Sure. I am sorry for the mistake.

**Comment by hammerstone18** [comment]
> 

ok thanks
